### ISSUE TITLE
Drop unneeded `nanorc` dependencies

### ISSUE THREAD

**Issue by jcfreeman2** [DUNE-DAQ/daq-release#415]
Today, I've dropped `nanorc` from the v5 development line. However, by a rough count there are about 60 different Python packages which `nanorc` imports from in various points of its codebase; whereas some of these are built-in packages, and some of these are used in Python code elsewhere in DUNE DAQ, there are likely some dependencies which will no longer be needed. In the interests of good hygiene, we should get rid of those.

**Comment by jcfreeman2** [comment]
Handled via https://github.com/DUNE-DAQ/daq-release/commit/32a9f08a74637924730ece8792b38731ae317519 on the `johnfreeman/update_externals` branch; see also the normal integration test performance when using a test nightly with the `nanorc` dependencies stripped, https://github.com/DUNE-DAQ/daq-release/actions/runs/12679174774.  This issue should be closed once `johnfreeman/update_externals` is merged into `develop`.

**Comment by jcfreeman2** [comment]
`johnfreeman/update_externals` has been merged into `develop` (by way of the `johnfreeman/transition_to_ext_v2.2` branch, which was merged yesterday in https://github.com/DUNE-DAQ/daq-release/pull/424) so I'm closing this.
